projects/removingExtraWhiteSpaces/_posts/2020-10-21-c.md

```diff
@@ -0,0 +1,80 @@
+---
+title: removingExtraWhiteSpaces in Every Language
+layout: default
+date: 2020-10-21
+last-modified: 2020-10-21
+featured-image:
+tags: [removingExtraWhiteSpaces, c]
+authors:
+    - [BHMBS Ibrahim](https://github.com/barhouum7)
```

Comment:
```suggestion
    - barhouum7
```

I'm not sure what you want your username to be, but you can add yourself to the _data/authors.yml.

Reply:
I'm sorry I didn't know about authors.yml